doc/discretization.rst

```diff
@@ -0,0 +1,312 @@
+\b{F}^{I}(\b{Q}_h^+)+\b{F}^{I}(\b{Q}_h^-)\right) - \frac{\lambda}
+{2}\left(\b{Q}_h^+ - \b{Q}_h^-\right)\b{n},
+$$
+where $\lambda$ is the characteristic max wave-speed of the fluid. Numerical fluxes
+which penalize the ''jump'' of the state $\left(\b{Q}_h^+ - \b{Q}_h^-\right)
+\b{n}$ act as an additional source of dissipation, which has a stabilizing effect
+on the numerics.
+
+Numerical viscous flux
+^^^^^^^^^^^^^^^^^^^^^^
+Take $\b{h}_v(\b{Q}_h^+, \b{\Sigma}_h^+, \b{Q}_h^-, \b{\Sigma}_h^-; \b{n})$ to be
+defined in a ''centered'' (average) way:
+$$
+\b{h}_v(\b{Q}_h^+, \b{\Sigma}_h^+, \b{Q}_h^-, \b{\Sigma}_h^-;
+\b{n}) = \frac{1}{2}\left(\b{F}^V(\b{Q}_h^+, \b{\Sigma}_h^+) +
+\b{F}^V(\b{Q}_h^-, \b{\Sigma}_h^-)\right)\cdot\b{n}
+$$
+
+Numerical gradient flux
+^^^^^^^^^^^^^^^^^^^^^^^
+And similarly for the gradient flux:
+$$
+\b{H}_s(\b{Q}_h^+, \b{Q}_h^- ; \b{n}) = \frac{1}{2}\left(
+\b{Q}_h^+ + \b{Q}_h^-\right)\b{n}
+$$
+
+Domain boundary considerations
+------------------------------
+
+What happens when $\partial E \cap \partial\Omega \neq \emptyset$?
+
+In DG, numerical fluxes are not only responsible for handling the flow of information
+between adjacent cells, but they also enforce information flow at the boundaries.
+
+We denote the *boundary fluxes* as $\b{h}^*_e(\b{Q}_{bc})$,
+$\b{h}^*_v(\b{Q}_{bc}$, $\b{\Sigma}_{bc})$, and
+$\b{H}^*_s(\b{Q}_{bc})$, where $\b{Q}_{bc}$, $\b{\Sigma}_{bc}$ denote
+boundary conditions imposed on the state, and the gradient of the state respectively.
+
+For all $\partial E \cap \partial\Omega$ there is no $+$ side to consider; just the
+interior state ($-$ side) and the prescribed boundary conditions $\b{Q}_{bc},
+\b{\Sigma}_{bc}$.
+
+Solid walls
+^^^^^^^^^^^
+
+Inviscid boundary flux
+""""""""""""""""""""""
+$\b{h}^*_e$ is equal to the (interior; - side) pressure contribution of
+$\b{F}^I(\b{Q}_{bc})\cdot\b{n}$
+(since $\b{V}\cdot\b{n} = 0$).
+
+Viscous boundary flux
+"""""""""""""""""""""
+$$
+\b{h}^*_v(\b{Q}_{bc}, \b{\Sigma}_{bc}) = \b{F}_V(\b{Q}_{bc},
+\b{\Sigma}_{bc})\cdot\b{n},
+$$
+where $\b{Q}_{bc}$ are the same values used to prescribe $\b{h}^*_e$.
+
+
+Gradient boundary flux
+""""""""""""""""""""""
+If there are no conditions on $\nabla\b{Q}\cdot\b{n}$, then:
+$$
+\b{\Sigma}_{bc} = \b{\Sigma}_h^-.
+$$
+Otherwise, $\b{\Sigma}_{bc}$ will need to be modified accordingly.
+
+Inflow/outflow boundaries
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Inviscid boundary flux
+""""""""""""""""""""""
+$$
+\b{h}^*_e(\b{Q}_{bc}) = \b{h}_e(\b{Q}_{bc}, \b{Q}^-_{h};
+\b{n}).
+$$
+
+Viscous boundary flux
+"""""""""""""""""""""
+$$
+\b{h}^*_v = \b{h}_v(\b{Q}_{bc}, \b{\Sigma}_h^-, \b{Q}_h^-,
+\b{\Sigma}_h^-; \b{n}),
+$$
+where $\b{Q}_{bc}$ are the same values used for $\b{h}^*_e$.
+
+
+Gradient boundary flux
+""""""""""""""""""""""
+$\b{Q}_{bc}$ is also used to define the gradient boundary flux:
+$$
+\b{H}^*_s(\b{Q}_{bc}) = \b{Q}_{bc}\b{n}.
+$$
+
+Second-order terms on the viscous RHS
```

Comment:
I'm not totally sure why this is here... it feels more like internal notes. It's also not discrete, which makes this seemingly a bit out of place in the discretization section?

Reply:
Let's remove it.

Reply:
Had trouble letting go of it I guess. (5e939db), (f826ea1)